core/Pos/Core/Script.hs

```diff
@@ -73,17 +50,3 @@ instance NFData a => NFData (PLCore.PatternF a)
 instance NFData a => NFData (PLTypes.TypeF a)
 instance NFData PLCore.PrimData
 instance NFData PLCore.Program
-
-instance Bi PLCore.Term => SafeCopy PLCore.Term where
```

Comment:
These have been moved into a different module, as GHC was complaining for some reason about not being able to find the `Bi` definition for those.